pkgs/development/python-modules/redis-om/default.nix

```diff
@@ -68,10 +65,7 @@ buildPythonPackage rec {
   # https://github.com/redis/redis-om-python/issues/532
   doCheck = false;
 
-  pythonImportsCheck = [
-    "aredis_om"
-    "redis_om"
```

Comment:
Upstream distributes both of these on PyPI. Maybe there's something wrong with our `preBuild`?